### PR TITLE
Create definition and test: angular acceleration from angular velocity derivative

### PR DESCRIPTION
**Pull request by Voldoned**

### CODE REVIEW

File: symplyphysics/definitions/angular_acceleration_is_angular_velocity_derivative.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+def print_law() -> str:
+    return print_expression(definition)
+
+
+@validate_input(angular_velocity_start_=angular_velocity, angular_velocity_end_=angular_velocity, moving_time_=time)
+@validate_output(angular_acceleration)
+def calculate_angular_acceleration(angular_velocity_start_: Quantity,
+    angular_velocity_end_: Quantity, moving_time_: Quantity) -> Quantity:
+    #HACK: SymPy angles are always in radians
+    angular_acceleration_ = time * (angular_velocity_end_ - angular_velocity_start_) / moving_time_
```
Comment (blackyblack) [inline symplyphysics/definitions/angular_acceleration_is_angular_velocity_derivative.py:33]: This is velocity function.

File: test/definitions/angular_acceleration_is_angular_velocity_derivative_test.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    Args = namedtuple("Args", ["w0", "w1", "t"])
+    return Args(w0=w0, w1=w1, t=t)
+
+
+def test_basic_acceleration(test_args):
+    result = angular_acceleration_def.calculate_angular_acceleration(test_args.w0, test_args.w1,
+        test_args.t)
+    assert SI.get_dimension_system().equivalent_dims(result.dimension, 1 / (units.time ** 2))
+    result_velocity = convert_to(result, angular_acceleration_def.definition_units_SI).evalf(2)
+    assert result_velocity == approx(0.63, 0.01)
```
Comment (blackyblack) [inline test/definitions/angular_acceleration_is_angular_velocity_derivative_test.py:33]: It's hard to decide if 0.63 equals to pi/5 from description. Use pi/5 here.

File: test/definitions/angular_acceleration_is_angular_velocity_derivative_test.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    units,
+    convert_to,
+    Quantity,
+    SI,
+    angle_type, dimensionless
+)
+from symplyphysics.definitions import angular_acceleration_is_angular_velocity_derivative as angular_acceleration_def
+
+# Description
+## Suppose that the object starts rotating with some angular acceleration. After 5 seconds, it rotates at 180 degrees per second.
```
Comment (blackyblack) [inline test/definitions/angular_acceleration_is_angular_velocity_derivative_test.py:15]: Use pi radians per second, to conform to the code below.

File: symplyphysics/definitions/angular_acceleration_is_angular_velocity_derivative.py (hunk 1 above)
Comment (Voldoned) [reply symplyphysics/definitions/angular_acceleration_is_angular_velocity_derivative.py:33]: Why?

File: symplyphysics/definitions/angular_acceleration_is_angular_velocity_derivative.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    return print_expression(definition)
+
+
+@validate_input(angular_velocity_start_=angular_velocity, angular_velocity_end_=angular_velocity, moving_time_=time)
+@validate_output(angular_acceleration)
+def calculate_angular_acceleration(angular_velocity_start_: Quantity,
+    angular_velocity_end_: Quantity, moving_time_: Quantity) -> Quantity:
+    #HACK: SymPy angles are always in radians
+    angular_velocity_function = time * (angular_velocity_end_ - angular_velocity_start_) / moving_time_
+    applied_definition = definition.subs(angular_acceleration(time), angular_velocity_function)
```
Comment (blackyblack) [inline symplyphysics/definitions/angular_acceleration_is_angular_velocity_derivative.py:34]: You should substitute velocity with velocity function.

Comment (Voldoned) [reply symplyphysics/definitions/angular_acceleration_is_angular_velocity_derivative.py]: Thanks!

**Review by blackyblack — APPROVED** [review]

**Pull request merged**